conda_recipe_manager/commands/bump_recipe.py

```diff
@@ -0,0 +1,59 @@
+"""
+:Description: CLI for bumping build number in recipe files.
+"""
+
+from __future__ import annotations
+
+import sys
+from pathlib import Path
+from typing import cast
+
+import click
+
+from conda_recipe_manager.commands.utils.print import print_err
+from conda_recipe_manager.commands.utils.types import ExitCode
+from conda_recipe_manager.parser.recipe_parser import RecipeParser
+from conda_recipe_manager.types import JsonPatchType
+
+
+@click.command(short_help="Bump recipe build version.")
+@click.argument("recipe_file_path", type=click.Path(exists=True, path_type=str))
+@click.option(
+    "--build-num",
+    is_flag=True,
+    help="bump the build number by 1.",
+)
+def bump_recipe(recipe_file_path: str, build_num: bool) -> None:
+    """
+    Bump recipe version.
```

Comment:
```suggestion
    Bumps a recipe to a new version. In other words, this tool aims to automatically update recipe files.
```

Reply:
@schuylermartin45 , I am only changing it to : "Bumps a recipe to a new version."
If you feel strongly about the `In other words, this tool aims to automatically update recipe files.` comment, let me know and we can discuss.